snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/tryparseex1.cs

```diff
@@ -12,13 +12,11 @@ public static void Main()
                                originalGuid.ToString("X") };
       
       // Parse each string representation.
-      Guid newGuid;
       foreach (var stringGuid in stringGuids) {
-         if (Guid.TryParse(stringGuid, out newGuid))
-            Console.WriteLine("Converted {0} to a Guid", stringGuid);
+         if (Guid.TryParse(stringGuid, out var newGuid))
```

Comment:
I think this change is **incorrect**. var is used when declaring variables. However, I still didn't test this in an IDE.

Reply:
I did test this on https://try.dot.net/. Check this: https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7#out-variables

I'd use discards since that variable is not used but try.net didn't like that.

Reply:
@mairaw, Thanks, didn't know this feature of C# 7.

I'll open another PR for the other occurrences of `var` that doesn't follow the guidelines like:

```csharp
var g = Guid.NewGuid();
```

Reply:
@Youssef1313 

```csharp
var g = Guid.NewGuid();
```

To me, that example is clear. I would be be very surprised if a method `Guid.NewGuid()` returned anything other than a `Guid`. When I reviewed the changes in this PR, I didn't find uses of `var` that were unclear. I don't think another PR is needed here.